### ISSUE TITLE
[Bug] Sorting only occurs after selecting sorting mode

### ISSUE THREAD

**Issue by HisMaj3sty** [FurryLord/TO-DO-list#6]
**Steps to reproduce:**
1. Select sorting by date
2. Add Task1 with date 15.01.2023
3. Add Task2 with date 13.01.2023

**Expected result:**
Task2 is placed before Task1

**Actual result:**
Task2 is placed after Task1

**Comments:**
If sorting by date is selected again the sorting is applied correctly

**Suggested action:**
Apply the sorting every time after a new task is added
